content/news/Pan-STARRS-data_HLSP.md

```diff
@@ -0,0 +1,12 @@
+Title: Release of Pan-STARRS C16 and C17 sparse time series photometry
+Date: 2018-09-25 09:00
+Author: Michael Gully-Santiago
+
+Pan-STARRS observed the K2 Campaign 16 and 17 fields contemporaneously from the ground.  All of the data are available as a [new High Level Science Product](https://archive.stsci.edu/prepds/k2sne/) at the Mikulski Archive for Space Telescopes (MAST).  The Pan-STARRS data include sparse *griz* time-series photometry of the entire 115 square degree K2 field of view, acquired in 7 square degree mosaics.  The number of time samples ranges from about 4 to 35 nights depending on the band and campaign.  K2 Project Scientist Jessie Dotson and collaborators have also published [a new Research Note of the AAS](http://iopscience.iop.org/journal/2515-5172) describing the motivation for contemporaneous observations, date ranges, per-band photometric cadences, and field center positions.
+
+
+<a href="images/news/projections_c16.png"><img class="img-responsive" style="max-width:500px;" src="images/news/projections_c16.png" alt="C16 Pan-STARRS Field"></a>
+
+<a href="images/news/projections_c17.png"><img class="img-responsive" style="max-width:500px;" src="images/news/projections_c17.png" alt="C17 Pan-STARRS Field"></a>
+
+The observations have already been extremely useful for identifying and characterizing transient phenomena such as supernovae.  The C16 field also includes the star clusters M44 and M67, for which polychromatic photometry can assess stellar activity and starspot physical properties for example.  The entire dataset can be downloaded, or individual targets queried.
```

Comment:
Let's remove the word `extremely` (I like to avoid the word in scientific prose).